src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetails.tsx

```diff
@@ -83,7 +83,7 @@ export const FulfillmentDetails: FC<FulfillmentDetailsProps> = ({
 
   /*
    * Re-save fulfillment details on load if they are already saved
-   * and shipping quotes need refreshing for new address mode only
+   * and shipping quotes need refreshing for new address mode only ?? Is this correct?
```

Comment:
I should probably stare at this until I can delete the new text here.

Reply:
Yea - the way I read it here it's not really checking for `new mode` below. It seems to always refresh artsy quotes and update fulfillment details  if `Ship` is selected and works is artsy shipped to a given address

Reply:
I read through this and thought about how it works. Updated here, can share more details in slack.